proto/sentry_protos/snuba/v1/endpoint_trace_item_details.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+// This is a request to the TraceItemDetails endpoint,
+// it is used to query for TraceItems (ex. spans or logs)
+// it returns all of the information for a specific trace item.
+message TraceItemDetailsRequest {
+  RequestMeta meta = 1;
+
+  //required: the ID (hex string) of the item you are looking for
+  string item_id = 2;
+  //OPTIONAL: improves performance if included
+  string trace_id = 3;
```

Comment:
Should this be the filter as per the notion https://www.notion.so/sentry/EAP-API-Design-1198b10e4b5d80faa5e9dcec47a9c12b?pvs=4#11b8b10e4b5d80e78ad8f13f9e58c9ff 
```suggestion
  repeated TraceItemFilter filter = 3;
```

Reply:
Let's indeed leave it as a filter. If we make it required later only, we'll promote it.